Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,283 @@
+<img src="../Documentation/Images/BoundingBox/MRTK_BoundingBox_HandleStyles2.png" width="450">
+
+**Clipping Example**
+
+The ClippingExamples scene, demonstates using the MRTK Standard Shader’s new clipping feature.
+
+**CoreServices**
+
+The [CoreServices](xref:Microsoft.MixedReality.Toolkit.CoreServices) static class works in conjunction with the
+MixedRealityServiceRegistry to provide applications with a fast and convenient mechanism to acquire instances of core
+services (ex: Input System).
+
+To access extension service instances, use `MixedRealityServiceRegistry.TryGetService<T>`.
+
+**Default HoloLens (1st gen) profile**
+
+We have added a new profile for HoloLens (1st gen) development that includes some of the
+recommended MRTK configurations for best performance.
+
+To configure your application for HoloLens (1st gen) optimized settings, set the
+Mixed Reality Toolkit's **Active Profile** to **DefaultHoloLens1ConfigurationProfile**.
+
+![Default HoloLens (1st gen) Configuration Profile](./Images/ReleaseNotes/DefaultHoloLens1ConfigurationProfile.png)
+
+**Dependency Window**
+
+A [Dependency Window](Tools/DependencyWindow.md) has been added which displays how assets reference and depend on each other.
+This tool can be used to easily determine which assets within a project are not being used.
+
+**Extension Service Creation Wizard**
+
+Making the transition from singletons to services can be difficult. The [Extension Service Creation Wizard](Tools/ExtensionServiceCreationWizard.md)
+supplements documentation and sample code by enabling devs to create new services easily
+
+**Fingertip Cursor**
+
+The fingertip cursor has been updated to better match the HoloLens 2 shell.
+
+**Hand Menu Graduated from Experimental**
+
+The HandConstraint and HandConstraintPalmUp solvers are now an official feature (no longer experimental) and have improved documentation.
+
+The HandConstraintPalmUp solver now has a toggle to enforce the hand’s fingers are coplanar before activating.
+
+**HoloLens 2 Button**
+
+- Improved many visual details to match the HoloLens 2 shell including
+    - Compressing visuals
+    - Far interaction support
+    - Focus highlight
+    - Shader effects
+- HoloLens 2 style Round Button has been added
+
+**HoloLens 2 Shell Parity**
+
+This release updates the MRTK to better mirror the featires. behaviors and visuals of the HoloLens 2 shell experience. This [GitHub](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/4200) issue describes the changes.
+
+**IMixedRealityRaycastProvider**
+
+The Input System was modified to add a reference to an IMixedRealityRaycastProvider.
+
```

Comment:
The input system was updated to allow for raycast customization, via custom implementations of IMixedRealityRaycastProvider.

It's important to call out here not what happened, but how it impacts customers and what they can do with it.

The existing sentence is kinda like "oh cool, it has a reference!" but really it's more like "as a customer, you can now do XYZ more because of this"

Reply:
updated. thanks!